modules/monitoring-enabling-a-separate-alertmanager-instance-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * monitoring/enabling-alert-routing-for-user-defined-projects.adoc
+
+:_content-type: PROCEDURE
+[id="monitoring-enabling-a-separate-alertmanager-instance-for-user-defined-projects_{context}"]
+= Enabling a separate Alertmanager instance for user-defined projects
+ 
+In some clusters, you might not want to use the default platform instance of Alertmanager for user-defined alerts. 
+In these cases, you can optionally enable a dedicated instance of Alertmanager to send alert notifications for user-defined projects in a cluster.
```

Comment:
The juxtaposition of "user-defined alerts" in line 9 and "alert notifications for user-defined projects in a cluster" in line 10 is raises questions. Are the two equivalent or distinct?

Reply:
Edited to read `In some clusters, you might not want to use the default platform instance of Alertmanager to send alert notifications for user-defined projects. 
In these cases, you can optionally enable a dedicated instance of Alertmanager to send alerts for these projects.`